docs/docs/coverage/language/index.md

```diff
@@ -34,6 +34,7 @@ On the other hand, when the target is a post-build artifact, like a container im
 | [.NET](dotnet.md)    | packages.lock.json                                                                         |     ✅     |     ✅      |       ✅        |       ✅        |
 |                      | packages.config                                                                            |     ✅     |     ✅      |       ✅        |       ✅        |
 |                      | .deps.json                                                                                 |     ✅     |     ✅      |       ✅        |       ✅        |
+|                      | *Packages.props[^11]                                                                       |     ✅     |     ✅      |       ✅        |       ✅        |
```

Comment:
We divide all language files into “before build” and “after build” - https://aquasecurity.github.io/trivy/v0.47/docs/coverage/language/#supported-languages
Perhaps we should only scan Packages.props files in fs/repo mode?

Reply:
Is there any reason why we should not also support it in Image?

Reply:
By the way, same thing could be said about `packages.lock.json`. I think there will be mostly .dll files in post-build.

Reply:
Also, If I use my image for development purposes (For example I pre-build an image with the source code to run unit tests or some static code analysis), it is actually make sense to look for these artifacts

Reply:
You are right.
Looks like we need to use pre-build for all supported NuGet files.
But then NuGet packages will not be scanned in `image` mode.

Okay, let's use same logic for `*Packages.props`.